### PR TITLE
[Release-Only] Branch Cut - Bump vision Version 0.16.0

### PR DESCRIPTION
**Pull request by huydhn**
Prepare the branch cut for vision

### CODE REVIEW

File: .github/workflows/build-conda-linux.yml [hunk 1]
```diff
@@ -15,12 +15,13 @@ on:
...
   generate-matrix:
-    uses: pytorch/test-infra/.github/workflows/generate_binary_build_matrix.yml@main
+    uses: pytorch/test-infra/.github/workflows/generate_binary_build_matrix.yml@release/2.1
     with:
       package-type: conda
       os: linux
       test-infra-repository: pytorch/test-infra
-      test-infra-ref: main
+      test-infra-ref: release/2.1
+      channel: test
```
Comment (osalpekar) [inline .github/workflows/build-conda-linux.yml:24]: This is set automatically, so no need to explicitly define.

**Review by osalpekar — APPROVED** [review]
LGTM

File: .github/workflows/build-conda-linux.yml (hunk 1 above)
Comment (huydhn) [reply .github/workflows/build-conda-linux.yml:24]: You mean the `ref` branch or the `channel`?  I'm trying to see which one is set automatically on https://github.com/pytorch/test-infra/blob/main/.github/workflows/generate_binary_build_matrix.yml

Comment (atalman) [reply .github/workflows/build-conda-linux.yml:24]: normally channel: test, but in our current use case it does not matter, test config is the same as nightly

**Pull request merged**